tests/openapi/test_strictmode.py

```diff
@@ -1630,3 +1647,109 @@ def scroll_request(condition_count: int):
     # Operation requires 7 tokens (1 for the request and one per filter)
     assert response.status_code == 429
     assert "Read rate limit exceeded: Operation requires 7 tokens" in response.json()['status']['error']
+
+
+def test_strict_mode_fullscan_multivector(collection_name):
+    # Clear collection to not depend on other tests
+    response = request_with_validation(
+        api='/collections/{collection_name}',
+        method="DELETE",
+        path_params={'collection_name': collection_name},
+    )
+    assert response.ok
```

Comment:
How about just using a different collection name?

Reply:
If I create a collection with a different name, it won't be automatically deleted at the end of test (success or failure).

Reply:
We can add it in the fixture up-top:

```
@pytest.fixture(autouse=True)
def setup(collection_name):
    basic_collection_setup(collection_name=collection_name)
    yield
    drop_collection(collection_name=collection_name)
```

Reply:
The test uses multivectors which are not setup by `basic_collection_setup`.
I think I am missing something here, what is your concern? :)